### PR TITLE
Documentation: Add missing configuration variables for METIS

### PR DESCRIPTION
**Pull request by maxGimeno**
## Summary of Changes

Add some information to the Summary page of the Getting Started documentation.
## Release Management

* Affected package(s):Documentation

### CODE REVIEW

File: Documentation/doc/Documentation/advanced/Configuration_variables.txt [hunk 1]
```diff
@@ -85,6 +85,7 @@ and will serverly limit performances.
 | Variable              | Description                                                           | Type        | %Default Value |
 | :-                    | :-                    | :-   |  :-            |
 | `CGAL_DIR`            | Full-path to the binary directory where \cgal was configured          |Either CMake or Environment | none |
+| `Qt5_DIR`             | Full-path to the Qt cmake directory  |CMake| platform-dependent|
```
Comment (lrineau) [inline Documentation/doc/Documentation/advanced/Configuration_variables.txt]:
```suggestion
| `Qt5_DIR`             | Full-path to the Qt cmake directory                                   |Either CMake or Environment | none |
```

**Comment by maxGimeno** [comment]
Successfully tested in https://cgal.geometryfactory.com/CGAL/testsuite/results-5.1-Ic-88.shtml

**Comment by lrineau** [comment]
The milestone of this PR is 5.0.3, but the base branch is `master`. Which one is right?

**Comment by maxGimeno** [comment]
> The milestone of this PR is 5.0.3, but the base branch is `master`. Which one is right?

5.0.3

**Comment by maxGimeno** [comment]
Successfully tested in https://cgal.geometryfactory.com/CGAL/testsuite/results-5.1-Ic-98.shtml

**Pull request merged**